crates/core/examples/challenge.rs

```diff
@@ -1,6 +1,6 @@
 use rand::Rng;
-use ya_relay_server::challenge::{self, PREFIX_SIZE, SIGNATURE_SIZE};
-use ya_relay_server::crypto::{CryptoProvider, FallbackCryptoProvider};
+use ya_relay_core::challenge::{self, PREFIX_SIZE, SIGNATURE_SIZE};
```

Comment:
In the near future we could reduce challenge difficulty and move this example to tests

Reply:
great idea!